main.py

```diff
@@ -2,6 +2,28 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
```

Comment:
Your code is repeating itself. Imagine that you have a file and you need to write all the information in the database. You will have to perform the same operation many times, which is not optimal.  I recommend putting all the inputs into a tuple  `("Western", "Action", "Dramma")` and then iterating over them, and then adding the values to the corresponding table columns.